src/DGtal/geometry/doc/moduleAlphaThickSegmentReco.dox

```diff
@@ -0,0 +1,249 @@
+with the possibility to take into accounts some noise. Moreover the
+segment can be detected from points which are not necessary connected
+or with floating coordinates.
+
+@note The proposed implementation is mainly a backport from
+ [ImaGene](https://gforge.liris.cnrs.fr/projects/imagene) with some
+ various refactoring.
+
+
+[TOC]
+
+
+\section moduleAlphaThickSegmentRecoIntro  Alpha Thick Segment
+
+The first definition of \b blurred \b segment  \cite debledFuzzy2005
+relies on the definition of the digital straight line (@ref
+moduleArithDSSReco):
+
+A set \f$ \mathcal{S}_f \f$ of consecutive points (\f$ \mathcal{S}_f
+\geq 2 \f$) of an 8-connected curve is a blurred segment with order
+\f$ d \f$ if there exists a discrete line \f$ \mathcal{D}(a,b, \mu,
+\omega) \f$ such that all points of \f$ \mathcal{S}_f \f$ belong to
+\f$ \mathcal{D} \f$ and \f$ \frac{\omega}{max(|a|,|b|) \leq d
+} \f$. The line \f$ D \f$ is said bounding for \f$ \mathcal{S}_f \f$.
+
+
+Following this definition, the authors also propose an incremental
+linear time algorithm to segment a digital curve into blurred segments
+of order \f$ d \f$. The main drawback of such a decomposition is the
+non optimality (it implies over segmentation).  Then the author
+introduce the notion blurred segment of width \f$ \nu \f$ 
+\cite debledrennessonblurred2005. The width \f$ \nu \f$ is associated to the
+isothetic thickness of a set of points, ie. the minimum value between
+its vertical and horizontal thicknesses (see figure below).
+
+@image html convexHullVT_HT.png "The isothetic thickness of this convexhull is the minimum between its vertical (VT) and horizontal (HT) thickness. The points P,Q and S are the antipodal pair." 
+@image latex convexHullVT_HT.png "The isothetic thickness of this convexhull is the minimum between its vertical (VT) and horizontal (HT) thickness. The points P,Q and S are the antipodal pair." width=5cm
+
+From this notion of isothetic thickness follows the \b blurred \b segment definition \cite DebledRennessonBlurred2005 :
+
+A bounding line of a digital set \f$ \mathcal{S}_b \f$ is said optimal
+if its isothetic thickness is equal to the isothetic thickness of the
+convex hull \f$ conv(\mathcal{S}_b) \f$. \f$ \mathcal{S}_b \f$ is a
+blurred segment of width \f$ \nu \f$ if and only if its optimal
+bounding line has an isothetic distance less or equal to \f$ \nu \f$.
+
+
+The actual implementation proposed here, relies on the convex hull
+computed incrementally with the melkman algorithm in linear time \cite melkmanLine1987.  
+It allows to add point on the front of the current
+segment and to update the value of vertical/horizontal thickness with
+a coast of \f$O( log\ n) \f$ (instead \f$O(n\ log\ n) \f$ thanks to the
+Buzer optimization \cite lilianComputing2007 ).  Note that the
+convex hull maintenance in linear time with point subtraction
+proposed by Buzer \cite lilianComputing2007 is not yet implemented.
+
```

Comment:
You call alpha thick segments "blurred segments" till here. Then you change. Perhaps you should put right at their definition that they will be called alpha-thick segments afterwards.